tests/kafkatest/tests/core/transactions_test.py

```diff
@@ -218,8 +225,9 @@ def setup_topics(self):
     @cluster(num_nodes=9)
     @matrix(failure_mode=["hard_bounce", "clean_bounce"],
             bounce_target=["brokers", "clients"],
-            check_order=[True, False])
-    def test_transactions(self, failure_mode, bounce_target, check_order):
+            check_order=[True, False],
+            use_group_metadata=[True, False])
```

Comment:
The changes in transactions_test are mainly for compatibility. Nothing should be behaving differently